website_docs/manual.md

```diff
@@ -7,9 +7,88 @@ aliases: [/docs/instrumentation/go/instrumentation, /docs/instrumentation/go/man
 
 Instrumentation is the process of adding observability code to your application. There are two general types of instrumentation - automatic, and manual - and you should be familiar with both in order to effectively instrument your software.
 
+## Initializing a Tracer
+
+To create spans, you'll need to initialize a tracer first. Some important things to consider:
+
+* Tracers should live at the module level and have a lifecycle congruent to an app's lifecycle
+* There should only be one tracer per module
+* Tracers need to be shut down when a program ends
+* You'll need an [exporter]({{< relref "exporting_data" >}}) to create a Tracer
+* You'll need to create Resources at trace initialization time
+
+First, ensure you have the right packages installed:
+
+```
+go get go.opentelemetry.io/otel \
+  go.opentelemetry.io/otel/trace \
+  go.opentelemetry.io/otel/sdk \
+```
+
+Ensure you have an [exporter]({{< relref "exporting_data" >}}) configured, then
+
+```go
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.7.0"
+	"go.opentelemetry.io/otel/trace"
+)
+
+var (
+	tracer trace.Tracer
+)
+
+func newExporter(ctx context.Context)  /* (someExporter.Exporter, error) */ {
+	// Your preferred exporter: console, jaeger, zipkin, OTLP, etc.
+}
+
+func newTraceProvider(exp *otlptrace.Exporter) *sdktrace.TracerProvider {
+    // The service.name attribute is required.
+	resource :=
+		resource.NewWithAttributes(
+			semconv.SchemaURL,
+			semconv.ServiceNameKey.String("ExampleService"),
+		)
+
+	return sdktrace.NewTracerProvider(
+		sdktrace.WithBatcher(exp),
+		sdktrace.WithResource(resource),
+	)
+}
+
+func main() {
+	ctx := context.Background()
+
+	exp, err := newExporter(ctx)
+	if err != nil {
+		log.Fatalf("failed to initialize exporter: %v", err)
+	}
+
+	// Create a new tracer provider with a batch span processor and the given exporter.
+	tp := newTraceProvider(exp)
+
+	// Handle shutdown properly so nothing leaks
+	defer func() { _ = tp.Shutdown(ctx) }()
+
+	otel.SetTracerProvider(tp)
+
+	// Finally, set the tracer that can be used throughout the app
```

Comment:
```suggestion
	// Finally, set the tracer that can be used for this package
```

It might be useful to have additional examples for acquiring a `Tracer` from the global `TracerProvider` in an instrumentation package and/or from the `TracerProvider` associated with a `Span` extracted from a context.